models/common data models/recharge__line_item_enhanced.sql

```diff
@@ -0,0 +1,201 @@
+with charge_line_items as (
+
+    select * 
+    from {{ var('charge_line_item')}}
+
+), charges as (
+
+    select * 
+    from {{ var('charge') }}
+
+), charge_shipping_lines as (
+
+    select
+        charge_id,
+        sum(price) as total_shipping
+    from {{ var('charge_shipping_line') }}
+    group by 1
+
+{% if var('recharge__using_checkout', false) %}
+), checkouts as (
+
+    select *
+    from {{ var('checkout') }}
+
+{% endif %}
+
+), addresses as (
+
+    select * 
+    from {{ var('address') }}
+
+), customers as (
+
+    select * 
+    from {{ var('customer') }}
+
+), subscriptions as (
+
+    select *
+    from {{ var('subscription_history') }} 
+    where is_most_recent_record
+
+), enhanced as (
+    select
+        charge_line_items.charge_id as header_id,
+        charge_line_items.index as line_item_id,
```

Comment:
Very small request, but so far the datatypes for these two fields in the other models have been strings. In this platform they are integers. Would you be able to update the datatype for these to be strings to ensure uniformity of the model datatypes across platforms.